packages/integration-tests/src/entities/metadata.ts

```diff
@@ -14,9 +14,10 @@ export const authorMeta: EntityMetadata<Author> = {
   type: "Author",
   idType: "int",
   tagName: "a",
+  disableTags: false,
```

Comment:
while the config is for all, we can scope the option to be on a table level. lmk what you think